### ISSUE TITLE
view data for old table does not work in embedded sheet

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#326]
steps to reproduce:

1. Navigate to a straight table
2. Press enter or space
3. focuses cell inside straight table instead of toggling view data.
![Kapture 2021-10-08 at 17 28 14](https://user-images.githubusercontent.com/9249820/136583625-8d19bc5a-40b0-46fe-a727-e7229742af80.gif)
